### ISSUE TITLE
moving mouse pointer/cursor interferes with keyboard input

### ISSUE THREAD

**Issue by corpusc** [skycoin/cx#124]
both types of keyboard input get stomped when the mouse pointer/cursor is moving.
(both keyboard events, & keyboard polling, oddly enough)

**Comment by asahi3g** [comment]
Do you have a small .cx where i could repro the issue ?

**Comment by corpusc** [comment]
I just realized that although I updated & merged with upstream, I didn't actually build a new CX.EXE yet.  I was planning to do some work tonight, so I will update you with up to date CX.EXE results, and if it's still a problem, you will be able to get it from the "cx-games" repo.  In the "ridge-blaster" folder.  It's not up to date at the moment, so wait until you hear from me again.  I was just about to work on it right now but I just got notice that I'm being transitioned to another Upwork account at the moment.

I started a new game because the old CX version I'd been working with is too different (I exclusively used shorthand math, for 1 example), and I figured I should narrow the scope quite a bit from what I was trying to do in the old "snake" folder.  Which I will slowly delete as I port relevant pieces of it to Ridge Blaster.

**Comment by corpusc** [comment]
I'm sorry it took me awhile to get back on this.  I was hoping I would be transitioned to another account by now so I could combine checking on this with another work session.  But I'm still not on any Upwork account, so I figured I better follow up on this at least.  

I just updated & verified I'm using the latest CX.EXE, and this DOES work fine now!  There is no problem with either types of keyboard input!  

Side note: I do get some delay between my fast mouse button clicks before I see the OpenGL boxes rendered on the screen (in the polygon editor that I just started making inside the "cx-games/ridge-blaster" project).

But printing CLI text feedback, in response to events, is instantaneous.
